src/pages/landelijk/positief-geteste-mensen.tsx

```diff
@@ -28,35 +27,54 @@ import { FCWithLayout } from '~/components/layout';
 import { getNationalLayout } from '~/components/layout/NationalLayout';
 import { SEOHead } from '~/components/seoHead';
 import siteText from '~/locale/index';
+import {
+  NationalInfectedAgeGroups,
+  NationalInfectedAgeGroupsValue,
+} from '~/types/data.d';
+import { formatNumber, formatPercentage } from '~/utils/formatNumber';
+import { replaceKpisInText } from '~/utils/replaceKpisInText';
+import { formatDateFromSeconds } from '~/utils/formatDate';
+import { AgeGroupChartWrapper } from '~/domain/infected-people/age-group/age-group-chart-wrapper';
+import { replaceVariablesInText } from '~/utils/replaceVariablesInText';
+import { formatAgeGroupRange } from '~/domain/infected-people/age-group/age-group-chart';
 import {
   getNationalStaticProps,
   NationalPageProps,
 } from '~/static-props/nl-data';
 import { colors } from '~/style/theme';
-import { formatDateFromSeconds } from '~/utils/formatDate';
-import { formatNumber, formatPercentage } from '~/utils/formatNumber';
-import { replaceKpisInText } from '~/utils/replaceKpisInText';
 
 const text = siteText.positief_geteste_personen;
 const ggdText = siteText.positief_geteste_personen_ggd;
 
+function getAgeGroupExampleData(data: NationalInfectedAgeGroups) {
+  const ageGroupRange = '20-29';
+
+  const value = data.values.find(
+    (x: NationalInfectedAgeGroupsValue) => x.age_group_range === ageGroupRange
```

Comment:
is it really necessary to type the parameter? Isn't `data.values` not already typed?
```suggestion
    (x) => x.age_group_range === ageGroupRange
```